lib/seahorn/BvOpSem2Context.hh

```diff
@@ -102,6 +102,7 @@ private:
   std::shared_ptr<ZSolver<EZ3>> m_z3_solver;
 
   bool m_shouldSimplify = false;
+  std::set<Expr> m_addedToSolver;
```

Comment:
probably unordered set is better here

Reply:
or use ExprSet if don't want to make decision

Reply:
good point